CHANGELOG.md

```diff
@@ -1,5 +1,325 @@
+[gh-326]: https://github.com/WP-API/WP-API/issues/326
+[gh-293]: https://github.com/WP-API/WP-API/issues/293
+[gh-300]: https://github.com/WP-API/WP-API/issues/300
+[gh-294]: https://github.com/WP-API/WP-API/issues/294
+[gh-320]: https://github.com/WP-API/WP-API/issues/320
+[gh-364]: https://github.com/WP-API/WP-API/issues/364
+[gh-355]: https://github.com/WP-API/WP-API/issues/355
+[gh-359]: https://github.com/WP-API/WP-API/issues/359
+[gh-357]: https://github.com/WP-API/WP-API/issues/357
+[gh-374]: https://github.com/WP-API/WP-API/issues/374
+[gh-377]: https://github.com/WP-API/WP-API/issues/377
+[gh-376]: https://github.com/WP-API/WP-API/issues/376
+[gh-390]: https://github.com/WP-API/WP-API/issues/390
+[gh-391]: https://github.com/WP-API/WP-API/issues/391
+[gh-393]: https://github.com/WP-API/WP-API/issues/393
+[gh-384]: https://github.com/WP-API/WP-API/issues/384
+[gh-396]: https://github.com/WP-API/WP-API/issues/396
+[gh-405]: https://github.com/WP-API/WP-API/issues/405
+[gh-380]: https://github.com/WP-API/WP-API/issues/380
+[gh-410]: https://github.com/WP-API/WP-API/issues/410
+[gh-416]: https://github.com/WP-API/WP-API/issues/416
+[gh-397]: https://github.com/WP-API/WP-API/issues/397
+[gh-437]: https://github.com/WP-API/WP-API/issues/437
+[gh-438]: https://github.com/WP-API/WP-API/issues/438
+[gh-402]: https://github.com/WP-API/WP-API/issues/402
+[gh-309]: https://github.com/WP-API/WP-API/issues/309
+[gh-413]: https://github.com/WP-API/WP-API/issues/413
+[gh-411]: https://github.com/WP-API/WP-API/issues/411
+[gh-458]: https://github.com/WP-API/WP-API/issues/458
+[gh-281]: https://github.com/WP-API/WP-API/issues/281
+[gh-392]: https://github.com/WP-API/WP-API/issues/392
+[gh-335]: https://github.com/WP-API/WP-API/issues/335
+[gh-271]: https://github.com/WP-API/WP-API/issues/271
+[gh-462]: https://github.com/WP-API/WP-API/issues/462
+[gh-461]: https://github.com/WP-API/WP-API/issues/461
+[gh-441]: https://github.com/WP-API/WP-API/issues/441
+[gh-455]: https://github.com/WP-API/WP-API/issues/455
+[gh-486]: https://github.com/WP-API/WP-API/issues/486
+[gh-358]: https://github.com/WP-API/WP-API/issues/358
+[gh-473]: https://github.com/WP-API/WP-API/issues/473
+[gh-505]: https://github.com/WP-API/WP-API/issues/505
+[gh-499]: https://github.com/WP-API/WP-API/issues/499
+[gh-519]: https://github.com/WP-API/WP-API/issues/519
+[gh-401]: https://github.com/WP-API/WP-API/issues/401
+[gh-347]: https://github.com/WP-API/WP-API/issues/347
+[gh-378]: https://github.com/WP-API/WP-API/issues/378
+[gh-528]: https://github.com/WP-API/WP-API/issues/528
+[gh-595]: https://github.com/WP-API/WP-API/issues/595
+[gh-730]: https://github.com/WP-API/WP-API/issues/730
+[gh-481]: https://github.com/WP-API/WP-API/issues/481
+[gh-524]: https://github.com/WP-API/WP-API/issues/524
+[gh-933]: https://github.com/WP-API/WP-API/issues/933
+[gh-474]: https://github.com/WP-API/WP-API/issues/474
+[gh-985]: https://github.com/WP-API/WP-API/issues/985
```

Comment:
Can we order these please? (Assuming you're in Sublime, select and Edit > Sort Lines should do)